front-end-master/landing_page/index.html

```diff
@@ -25,8 +25,17 @@
     <link rel="stylesheet" media="screen and (max-device-width: 750px)" href="mobile.css" />
 
 	<!-- jquery -->
-	<script src="../common_files/js/jquery-3.1.1.js"></script>
-	<!-- <script src="index.js"></script> -->
+    <script src="../common_files/js/jquery-3.1.1.js"></script>
+    <!-- <script src="index.js"></script> -->
```

Comment:
I prefer extracting out the js into its own file and then referencing it there. Then we have a clean and distinct source of our JS code and won't need to rummage through html to find it.